keramik/src/deploy_images.md

```diff
@@ -40,53 +40,5 @@ docker buildx build --load -t keramik/runner:dev --target runner .
 kind load docker-image keramik/runner:dev
 ```
 
-Now we need to tell any resources that use the running image to use this new version of the runner.
-
-### Networks
-
-Network resources use the runner to bootstrap peers.
-Edit `small.yaml` to configure the image of the bootstrap runner.
-
-```yaml
-# small.yaml
----
-apiVersion: "keramik.3box.io/v1alpha1"
-kind: Network
-metadata:
-  name: small
-spec:
-  replicas: 2
-  # Use custom runner image for bootstrapping
-  bootstrap:
-    image: keramik/runner:dev
-    imagePullPolicy: IfNotPresent
-```
-
-You will then apply this to start the runner
-
-```shell
-kubectl apply -f small.yaml
-```
-
-### Simulations
-
-Simulations use the runner to run scenarios.
-Edit `basic.yaml` to configure the runner image for simulations.
-This will change the runner image for all jobs created by the simulation.
-
-
-```yaml
-# basic.yaml
----
-apiVersion: "keramik.3box.io/v1alpha1"
-kind: Simulation
-metadata:
-  name: basic
-  namespace: keramik-small
-spec:
-  scenario: ceramic-simple
-  users: 10
-  run_time: 4
-  image: keramik/runner:dev
-  imagePullPolicy: IfNotPresent
-```
```

Comment:
This section should not be deleted. Maybe it can live in a different section be this topic is not covered anywhere else. This section explains how to configure either the Network or Simulation resources to use a local build of the runner.

Reply:
Re-added in its own section now with a couple of links to it https://github.com/3box/keramik/pull/51/files#diff-68393f8f40dbae2e7f5a2f43d4cd93a4d9338f305017f5775a327825db9b7e23